### DISCUSSION TITLE
Display for ESP32 or Pico W

### DISCUSSION THREAD

**Discussion by zalves** [micropython/micropython#14021] [General]
Hello!

I'm looking for a recommendation of display 3.5 to 4 inch, to use with an ESP32 or Pico W that has an easy to use driver, it doens't have to have great performance, I just want to display some simple images. I've seen a few for sale, ILI94XX or ST77XX, but don't want to buy something that I can't use with micropython. No touch is needed.

anything nice and easy? :)

**Comment by ned-pcs** [comment]
Peter Hinch has several display drivers as part of his [`micropython-nano-gui`](https://github.com/peterhinch/micropython-nano-gui) project. See [here](https://github.com/peterhinch/micropython-nano-gui/blob/master/DRIVERS.md).

[Here is a list of displays](https://github.com/peterhinch/micropython-nano-gui/blob/master/DISPLAYS.md).

**Reply by zalves** [reply]
Thank you, I'll look into it. I've tried using small ST7789 but without success, something with the configuration as the display works with Arduino code.

**Comment by robert-hh** [comment]
There are drivers available for the ILI9341, ST7789 and SD1963. Personally if bought quite a few displays at www.buydisplay.com. The have a huge selection of diplays in all variants and sizes, deliver fast and the devices work.
